pkgs/tools/networking/opensnitch/go.mod

```diff
@@ -0,0 +1,16 @@
+module github.com/evilsocket/opensnitch/daemon
+
+go 1.14
```

Comment:
should this be 1.14?

Reply:
This is the upstream reference, I don't want to change it as long as it compiles with the latest go version (1.19)